nss/Makefile

```diff
@@ -1,11 +1,12 @@
-CC=gcc
-CFLAGS=-g -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
-LDFLAGS=$(shell pkg-config --libs glib-2.0)
+CC := gcc
+CFLAGS := -g -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
+LDFLAGS := $(shell pkg-config --libs glib-2.0)
+SCRIPTPATH ?= /usr/libexec/aad-auth
 
-all: clean libnss_aad.so.2 copy
+all: clean libnss_aad.so.2
 
 libnss_aad.so.2:
-	$(CC) -DSCRIPTPATH=\"/home/u/getent.sh\" passwd.c group.c shadow.c common.c  $(CFLAGS) $(LDFLAGS) -fPIC -shared -Wl,-soname,libnss_aad.so.2 -o libnss_aad.so.2
+	$(CC) -DSCRIPTPATH=\"$(SCRIPTPATH)\" passwd.c group.c shadow.c common.c $(CFLAGS) $(LDFLAGS) -fPIC -shared -Wl,-soname,libnss_aad.so.2 -o libnss_aad.so.2
```

Comment:
could we take all *.c in the `nss/` directory? I asked for a change on dboverride to Denison to not be built, so this will have no effect.